Tests/LibWeb/Text/expected/css/font-family-with-dash-prefix.txt

```diff
@@ -0,0 +1 @@
+serif, -apple-system
```

Comment:
Commit message: That is a funky typo! :family:

Reply:
Whooooooops 😅